docs/install/kubernetes.md

```diff
@@ -0,0 +1,225 @@
+ 
+## Prerequesites
+
+An up to date kubernetes cluster
+postgres instance 
+Redis instance
+
+## Assumptions
+
+You have already deployed Seqera Platform
+You will be deploying the application into to the wave namespace
+you have appropriate permissions to perform these actions. 
+
+
+## Create namespace
+
+```
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "wave"
+  labels:
+    app: wave-app
+```
+
+## Configure Wave
+
+The following configuration example is a base please see the the wave configuration reference for all configuration options and examples. 
+
+Change the following values to match your environment. 
+
+database url 
+username 
+password
+
+redis password
+
+Seqera Platform API Endpoint commonly your platform instance followed by `/api` depending on your deployment you may have choosen to expose this on a subdomain such as `api.platforminstance.example`
+
+```
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: wave-cfg
+  namespace: "wave"
+  labels:
+    app: wave-cfg
+data:
+  config.yml:
+    wave:
+      build:
+        enabled: false
+      mirror:
+        enabled: false
+      scan:
+        enabled: false
+      blobCache:
+        enabled: false
+      db:
+        uri: "jdbc:postgresql://localhost:5432/wave"
+        user: "postgres"
+        password: "postgres"
+    redis:
+      uri: "rediss://REPLACE_ME_WAVE_REDIS_URL:6379"
+    tower:
+      endpoint:
+        url: REPLACE_ME_TOWER_SERVER_URL
+    micronaut:
+      executors:
+        stream-executor:
+          type: FIXED
+          number-of-threads: 16
+      netty:
+        event-loops:
+          default:
+            num-threads: 64
+          stream-pool:
+            executor: stream-executor
+      http:
+        services:
+          stream-client:
+            read-timeout: 30s
+            read-idle-timeout: 5m
+            event-loop-group: stream-pool
+    loggers:
+      env:
+        enabled: false
+      bean:
+        enabled: false
+      caches:
+        enabled: false
+      refresh:
+        enabled: false
+      loggers:
+        enabled: false
+      info:
+        enabled: false
+      metrics:
+        enabled: true
+      health:
+        enabled: true
+        disk-space:
+          enabled: false
+        jdbc:
+          enabled: false
+```
+
+## Create deployment
+
+
+```
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wave
+  namespace: "wave"
+  labels:
+    app: wave-app
+spec:
+  selector:
+    matchLabels:
+      app: wave-app
+  template:
+    metadata:
+      labels:
+        app: wave-app
+    spec:
+      containers:
+        - image: REPLACE_ME_AWS_ACCOUNT.dkr.ecr.us-east-1.amazonaws.com/nf-tower-enterprise/wave:REPLACE_ME_WAVE_IMAGE_TAG
+          name: wave-app
+          ports:
+            - containerPort: 9090
+          env:
+            - name: MICRONAUT_ENVIRONMENTS
+              value: "postgres,redis,lite"
+            - name: WAVE_JVM_OPTS
```

Comment:
Are the default JVM opts suitable or should we provide specific opts for the customer.

Reply:
@llewellyn-sl I think we can remove this line for now.